### ISSUE TITLE
模型无法调用，求助

### ISSUE THREAD

**Issue by xinyidaren** [kendryte/kendryte-standalone-sdk#105]
模型生成代码（tensorflow2.0）:
input_image = tf.keras.layers.Input(shape=[128, 128, 3], name="input_image")
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(input_image)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Flatten()(center)
center = layers.Dense(16, activation='relu')(center)
center = layers.Dense(10, activation='softmax')(center)
model = Model(inputs=[input_image], outputs=[center])

model.summary() #一个很简单的tensorflow2.0模型

# model.save('mnist_cnn.h5')
converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
if not os.path.exists('./tflite_models'):
    os.mkdir('./tflite_models')
with open('./tflite_models/keras.tflite', 'wb') as f:
    f.write(tflite_model)

ncc转换版本为NNCase v0.2.0 Beta2

转换完的kmodel模型在程序调用时报错：

core dump: fault load
Cause 0x0000000000000005,EPC 0x00000000000008001b3b4
